src/core/analytics.js

```diff
@@ -1404,4 +1490,5 @@ export {
   getGroupTraits,
   startSession,
   endSession,
+  setAuthToken,
```

Comment:
do we need to update the loading snippet in order to buffer this one too?

Reply:
Yes